app/views/payments/_form.html.erb

```diff
@@ -0,0 +1,44 @@
+<%= form_with(model: payment) do |form| %>
+  <% if payment.errors.any? %>
+    <div style="color: red">
+      <h2><%= pluralize(payment.errors.count, "error") %>
+        prohibited this payment from being saved:</h2>
+
+      <ul>
+        <% payment.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div class='flex flex-col mt-3 gap-2'>
+    <div>
+      <%= form.text_field :name,
+                      placeholder: "Name",
+                      class:
+                        "w-full focus:ring-0 focus:outli border-l-0 border-r-0 border-gray-300 focus:outline-none placeholder-gray-400" %>
+    </div>
+
+    <div>
+      <%= form.text_field :amount,
+                      placeholder: "Amount",
+                      class:
+                        "w-full focus:ring-0 focus:outli border-l-0 border-r-0 border-gray-300 focus:outline-none placeholder-gray-400" %>
+    </div>
+
+    <div>
+      <%= form.collection_select :group_ids,
+                             current_user.groups,
+                             :id,
+                             :name,
+                             { selected: selected_group&.id },
+                             { multiple: true } %>
+    </div>
+
+    <div class='mx-auto text-gray-500 mt-4 mb-2 hover:cursor-pointer'>
+      <%= form.submit %>
+    </div>
+  </div>
+
```

Comment:
- [ ] According to the project requirements, when I create a payment, I should be redirected to the payments page but instead, I am redirected to this page 
![image](https://github.com/limavfabio/easy_pay/assets/45460192/c662be5c-2ec5-4610-9f53-fb6dfcb64a61)